src/app/frontend/common/components/sparkline/sparkline.html

```diff
@@ -0,0 +1,8 @@
+<svg viewBox="0,0 1,1"
+     preserveAspectRatio="none"
+     class="kd-sparkline">
+  <polygon
+     ng-if="sparklineCtrl.series"
```

Comment:
Fixed in a6b3d3ba377ac2f85691ba534766202d6d20dcc8 (but the conditional goes away in bcf427d86d62322dd0673e2bf6962658d00440d4 )